src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerOfAPerson = (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
```

Comment:
let's simplify it a bit
```suggestion
  const partner = sex === 'm'
      ? 'wife'
      : 'husband';
```